### ISSUE TITLE
Missing MSVCP140d.dll. How to slove it?

### ISSUE THREAD

**Issue by dev-kang** [DevSourceOfficial/Windows-Process-Killer#2]
<img width="1003" alt="image" src="https://github.com/DevSourceOfficial/Windows-Process-Killer/assets/24736642/c56bb263-33e3-443d-80a4-95f1a11810ba">

**Comment by DevSourceOfficial** [comment]
Try the windowsprocesskiller - Release.exe file, I just uploaded it. If that doesn't work let me know

**Comment by dev-kang** [comment]
This works fine, but is it possible to close the process by its name? Not by Windows title. In addition, is it possible to run the file directly through command line parameters?

**Comment by DevSourceOfficial** [comment]
yes there is, I will work on it when I get a few minutes but as far as doing it by process name and not window title I would probably need a couple of days to read the win api docs and see if I can find something. I will try and get the new version uploaded either tonight or tomorrow :)

**Comment by DevSourceOfficial** [comment]
I went ahead and made it so that you can input the title of the window and exit code via the command line arguments, however when inputting a process with a window title that includes spaces you need to enclose the whole title of the window in "" example: "Untitled - Notepad" I will try and figure out a way of doing it with the process name instead of the window title.
